README.md

```diff
@@ -92,11 +134,47 @@ You can check the status of the nodes using the EC2 Dashboard or by running the
 
     ./bin/muchos status
 
+## Launching an Azure cluster
+
+Before launching a cluster, you will need to complete the requirements for Azure above, clone the Muchos repo, and
+create [muchos.props] by making a copy of existing [muchos.props.example]. If you want to give others access to your
+cluster, add their public keys to a file named `keys` in your `conf/` directory.  During the setup of your cluster,
+this file will be appended on each node to the `~/.ssh/authorized_keys` file for the user set by the
+`cluster.username` property.  You will also need to ensure you have authenticated to Azure and set the target
+subscription using the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/manage-azure-subscriptions-azure-cli?view=azure-cli-latest).
+
+Muchos by default uses a CentOS 7 image that is hosted in the Azure marketplace. The Azure Linux Agent is already
+pre-installed on the Azure Marketplace images and is typically available from the distribution's package repository.
+Azure requires that the publishers of the endorsed Linux distributions regularly update their images in the Azure
+Marketplace with the latest patches and security fixes, at a quarterly or faster cadence. Updated images in the Azure
+Marketplace are available automatically to customers as new versions of an image SKU.
+
+Edit the values in the sections within [muchos.props] as below
+Under the `general` section, edit following values as per your configuration
+* `cluster_type = azure`
+* `cluster_user` should be set to the name of the administrative user
+* `proxy_hostname` (optional) is the name of the machine which has access to the cluster VNET
+
+Under the `azure` section, edit following values as per your configuration
+* `resource_group` and provide the same created in your Azure subscription for the cluster deployment
+* `vnet` and provide the same created in your Azure subscription for the cluster deployment
+* `subnet` and provide the same created in your Azure subscription for the cluster deployment
+* `numnodes` can be changed as per the cluster size
+* `vm_sku` can be specified from the available skus in the [selected Azure region](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage)
```

Comment:
I am confused by this property name and the link.  The props file and ansible code seems to use this as a [VM size](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes-general), however this documentation mentions images.  It seems like the image is hard coded in the ansible code.

Reply:
This is a good catch! That link is wrong. The link should refer to VM sizes, and not VM images. Currently the only supported image is Centos which as you mention is hard coded. Will follow up with a different PR that makes this configurable (similar to the ami config for EC2).

Reply:
> This is a good catch!

I found it during my attempt to document the properties.  I also learned a bit about resource groups during that documentation process.

Reply:
> Will follow up with a different PR that makes this configurable (similar to the ami config for EC2).

That would be useful.  When I run test on EC2 I often try different node types.

Reply:
👍 Done